Dialog/Brazilian Portuguese.txt

```diff
@@ -0,0 +1,54 @@
+modname_collabutils2= Collab Utils 2
+modoptions_collab_title= Collab Utils 2
+
+collabutils2_returntolobby= Voltar ao lobby
+collabutils2_restartspeedberry= Reiniciar morango-relógio
+
+collabutils2_returntolobby_confirm_title= VOLTAR AO LOBBY?
+collabutils2_returntolobby_confirm_note1= Se você escolher "Salvar", você poderá retornar ao lugar de onde você saiu
+collabutils2_returntolobby_confirm_note2= voltando ao mapa no lobby e escolhendo "Continuar".
+collabutils2_returntolobby_confirm_save= Salvar
+collabutils2_returntolobby_confirm_donotsave= Não salvar
+collabutils2_returntolobby_confirm_cancel= Cancelar
+
+collabutils2_speedberry_gold= Ouro:
+collabutils2_speedberry_silver= Prata:
+collabutils2_speedberry_bronze= Bronze:
+collabutils2_speedberry_timeranout= Acabou o tempo!
+
+collabutils2_chapterpanel_continue= Continuar
+collabutils2_chapterpanel_start= Recomeçar
+
+collabutils2_speedberry_optionmessage=
+	Você pode mudar a posição do cronômetro do morango-relógio em {# 9B3FB5}Opções de mod{#}.
+
+modoptions_collab_speedberrytimerposition= Posição do cronômetro do morango-relógio
+modoptions_collab_speedberrytimerposition_topleft= Canto superior esquerdo
+modoptions_collab_speedberrytimerposition_topcenter= Centro superior
+modoptions_collab_hidespeedberrytimerduringgameplay= Esconder cronômetro do morango-relógio enquanto joga
+
+modoptions_collab_displayendscreenforallmaps= Mostrar tela final para todos os mapas
+modoptions_collab_displayendscreenforallmaps_description= Necessário para verificar IL's no speedrun.com
+
+modoptions_collab_besttimetodisplayinjournal = Melhor tempo a ser mostrado no diário

```

Comment:
You should also remove that space here
```suggestion
modoptions_collab_besttimetodisplayinjournal= Melhor tempo a ser mostrado no diário

```

Reply:
I didn't even notice that! I've fixed that now

Reply:
Sorry, I forgot to send the file
[Brazilian Portuguese.txt](https://github.com/EverestAPI/CelesteCollabUtils2/files/7260224/Brazilian.Portuguese.txt)